program/mobilenets-armcl-opencl/armcl_graph_common.h

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright (c) 2018 cTuning foundation.
+ * See CK COPYRIGHT.txt for copyright details.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause.
+ * See CK LICENSE.txt for licensing details.
+ */
+
+#pragma once
+
+#if defined(ARMCL_18_05_PLUS)
+#include <arm_compute/graph.h>
+#include <arm_compute/graph/nodes/Nodes.h>
+#include <arm_compute/graph/backends/BackendRegistry.h>
+#include <arm_compute/graph/backends/CL/CLDeviceBackend.h>
+#else
+#include <arm_compute/graph/Graph.h>
+#include <arm_compute/graph/Nodes.h>
+#include <arm_compute/runtime/CL/CLScheduler.h>
+#endif
+
+#ifndef DATATYPE
+#define DATATYPE DataType::F32
+#endif
+
+inline void printf_callback(const char *buffer, unsigned int len, size_t complete, void *user_data) {
+  printf("%.*s", len, buffer);
+}
+
+inline void set_kernel_path() {
+  const char* kernel_path = getenv("CK_ENV_LIB_ARMCL_CL_KERNELS");
+  if (kernel_path) {
+    printf("Kernel path: %s\n", kernel_path);
+    arm_compute::CLKernelLibrary::get().set_kernel_path(kernel_path);
+  }
+}
+
+inline void init_armcl(arm_compute::ICLTuner *cl_tuner = nullptr) {
+  cl_context_properties properties[] =
+  {
+    CL_PRINTF_CALLBACK_ARM, reinterpret_cast<cl_context_properties>(printf_callback),
+    CL_PRINTF_BUFFERSIZE_ARM, static_cast<cl_context_properties>(0x100000),
+    CL_CONTEXT_PLATFORM, reinterpret_cast<cl_context_properties>(cl::Platform::get()()),
+    0
+  };
+  cl::Context::setDefault(cl::Context(CL_DEVICE_TYPE_DEFAULT, properties));
+  arm_compute::CLScheduler::get().default_init(cl_tuner);
+
+  // Should be called after initialization
+  set_kernel_path();
+
+#if defined(ARMCL_18_05_PLUS)
+  arm_compute::graph::backends::BackendRegistry::get()
+    .add_backend<arm_compute::graph::backends::CLDeviceBackend>(
+      arm_compute::graph::Target::CL);
+#endif
+}
+
+#if defined(ARMCL_18_05_PLUS)
+
+inline arm_compute::graph::DepthwiseConvolutionMethod str_to_dwsc_convolution_method(const char *method_name) {
```

Comment:
Takes some effort to translate `dwsc` :). Why not simply `depthwise`?

Reply:
This method is no more needed, removed.

Reply:
But the name was from ARM's `get_dwsc_node`